packages/react-workers/src/utils/sorting.js

```diff
@@ -0,0 +1,58 @@
+import { firstBy } from 'thenby';
+
+/**
+ * Apply sort structure to a collection of features
+ * @param {array} features
+ * @param {object} [sortOptions]
+ * @param {string | string[] | object[]} [sortOptions.sortBy] - One or more columns to sort by
+ * @param {string} [sortOptions.sortByDirection] - Direction by the columns will be sorted
+ */
+export function applySorting(features, { sortBy = [], sortByDirection = 'asc' } = {}) {
+  const isValidSortBy =
+    (Array.isArray(sortBy) && sortBy.length) || // sortBy can be an array of columns
+    typeof sortBy === 'string'; // or just one column
+
+  if (!isValidSortBy) {
+    return features;
+  }
+
+  const [firstSortOption, ...othersSortOptions] = normalizeSortByOptions({
+    sortBy,
+    sortByDirection
+  });
+
+  // Create sortFn
+  let sortFn = firstBy(...firstSortOption);
```

Comment:
Instead of add a comment, create a new `createSortFn`. It's just a suggestions, feel free to change or not :) 
Something like:
```js
function createSortFn({ sortBy, sortByDirection }) {
    const sortOptions = normalizeSortByOptions({
        sortBy,
        sortByDirection
     });
    return applySortOptions(sortOptions)
}
```

and call with 

```js
const sortFn = createSortFn({
    sortBy,
    sortByDirection
  });

```

Reply:
I also like it!

Reply:
Updated